blog/2023-06-08-Annoucing-Martin-Project-Manager

```diff
@@ -0,0 +1,14 @@
+---
+author: Advisory Committee
+title: "Announcing Martin Magdinier as the New Project Manager of OpenRefine"
+slug: 2023/06/08/Annoucing-Martin-Project-Manager
+---
+
+OpenRefine's advisory committee is delighted to officially announce Martin Magdinier ([github](https://github.com/magdmartin), [forum](https://forum.openrefine.org/u/Martin/summary)) as the new Project Manager of OpenRefine. 
+<!--truncate-->
+
+Having actively participated in the OpenRefine community for over a decade, Martin possesses invaluable insights into the governance changes underway. His expertise in grant writing, hiring, budget and roadmap management, and community building will be pivotal in driving sustainable growth for OpenRefine. Moreover, Martin's strong connection with diverse communities, including GLAM, civic, data journalist, and researcher communities, ensures that our project will embrace many perspectives.
+
+Martin's primary focus will be supporting the Advisory Committee and spearheading the establishment of the [Ambassador Council](https://forum.openrefine.org/t/proposition-to-create-openrefine-ambassador-council/462). He will also coordinate various outreach activities, liaising with partner organizations, including internship programs, funding sources, and Code for Science and Society, among other responsibilities. Martin will be working part-time with a commitment of 10 hours per week. 
+
+In light of assuming the Project Manager position, we regretfully accept his immediate resignation from the Advisory Committee. This decision ensures that only 49% of the Advisory Committee is occupied by OpenRefine's paid resources to avoid potential conflict of interest.
```

Comment:
```suggestion
In light of assuming the Project Manager position, we accept his immediate resignation from the Advisory Committee. This decision ensures that only 49% of the Advisory Committee is occupied by OpenRefine's paid resources to avoid potential conflict of interest.
```
No regrets to be had in such a case I think, since @magdmartin is not leaving the project

Reply:
Kudos.  @magdmartin has always struck a balance between oversight and transparency.  I do not doubt this will continue for the betterment of the Ambassador Council as well as the OpenRefine project.

Reply:
thank you @thadguidry